WeakAurasTemplates/TriggerTemplatesDataMists.lua

```diff
@@ -905,21 +1103,170 @@ templates.class.DRUID = {
+        { spell = 48505, type = "ability", buff = true, overlayGlow = true, usable = true }, -- Starfall
+        { spell = 50334, type = "ability", buff = true }, -- Berserk
+        { spell = 61336, type = "ability", buff = true }, -- Survival Instincts
+        { spell = 62606, type = "ability", charges = true, usable = true }, -- Savage Defense
+        { spell = 77758, type = "ability", debuff = true, requiresTarget = true }, -- Thrash
+        { spell = 77761, type = "ability", buff = true }, -- Stampeding Roar
+        { spell = 77764, type = "ability" }, -- Stampeding Roar
+        { spell = 78674, type = "ability", overlayGlow = true, usable = true }, -- Starsurge
+        { spell = 78675, type = "ability", usable = true }, -- Solar Beam
+        { spell = 80964, type = "ability", requiresTarget = true }, -- Skull Bash
+        { spell = 80965, type = "ability", requiresTarget = true }, -- Skull Bash
+        { spell = 88423, type = "ability" }, -- Nature's Cure
+        { spell = 88751, type = "ability", usable = true }, -- Wild Mushroom: Detonate
+        { spell = 102280, type = "ability", talent = 2 }, -- Displacer Beast
+        { spell = 102342, type = "ability", buff = true }, -- Ironbark
+        { spell = 102351, type = "ability", buff = true, talent = 6 }, -- Cenarion Ward
+        { spell = 102359, type = "ability", debuff = true, usable = true, talent = 8 }, -- Mass Entanglement
+        { spell = 102401, type = "ability", requiresTarget = true, usable = true, talent = 3 }, -- Wild Charge
+        { spell = 102543, type = "ability", buff = true }, -- Incarnation: King of the Jungle
+        { spell = 102547, type = "ability", buff = true }, -- Prowl
+        { spell = 102791, type = "ability" }, -- Wild Mushroom: Bloom
+        { spell = 102793, type = "ability", talent = 14 }, -- Ursol's Vortex
+        { spell = 102795, type = "ability", debuff = true, requiresTarget = true }, -- Bear Hug
+        { spell = 106737, type = "ability", charges = true, talent = 12 }, -- Force of Nature
+        { spell = 106832, type = "ability", requiresTarget = true, usable = true }, -- Thrash
+        { spell = 106839, type = "ability", requiresTarget = true, usable = true }, -- Skull Bash
+        { spell = 106898, type = "ability", buff = true, usable = true }, -- Stampeding Roar
+        { spell = 106922, type = "ability", buff = true }, -- Ursocs Macht
+        { spell = 106951, type = "ability" }, -- Berserk
+        { spell = 106952, type = "ability", usable = true }, -- Berserk
+        { spell = 108238, type = "ability", usable = true, talent = 5 }, -- Renewal
+        { spell = 108288, type = "ability", talent = 16 }, -- Heart of the Wild
+        { spell = 108291, type = "ability", buff = true, talent = 16 }, -- Heart of the Wild
+        { spell = 108292, type = "ability", buff = true, talent = 16 }, -- Heart of the Wild
+        { spell = 108294, type = "ability", buff = true, talent = 16 }, -- Heart of the Wild
```

Comment:
4 lines with 1 skill

Reply:
Indeed this is the case for Heart of the Wild. Interestingly enough 108288 doesn't seem to be a spell any of them cast (despite being the default spell ID for the talent) and 108293, guardian's, is missing.